freebsd/FreeBSDProcessList.c

```diff
@@ -411,7 +411,7 @@ static char* FreeBSDProcessList_readProcessName(kvm_t* kd, const struct kinfo_pr
 }
 
 static char* FreeBSDProcessList_readJailName(const struct kinfo_proc* kproc) {
-   char*  jname = NULL;
+   char*  jname;
```

Comment:
Keep initializion to `NULL` and only assign `"N/A""`if still at `NULL` upon trying to return.